frontend.js

```diff
@@ -0,0 +1,37 @@
+// var madlib = { "text": "  <input id='ad1' class='clr' type='text' name='noun' >"

```

Comment:
Please delete old, irrelevant comments. Comments like this should almost always be removed before opening a pull request. 

It would be good if you added some comments saying what the code is doing, but I won't require it for this merge.